examples/api/lib/widgets/nested_scroll_view/nested_scroll_view.3.dart

```diff
@@ -0,0 +1,197 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [NestedScrollView].
+
+void main() => runApp(const NestedScrollViewExampleApp());
+
+class NestedScrollViewExampleApp extends StatelessWidget {
+  const NestedScrollViewExampleApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: NestedScrollViewExample(),
+    );
+  }
+}
+
+class NestedScrollViewExample extends StatefulWidget {
+  const NestedScrollViewExample({super.key});
+
+  @override
+  State<NestedScrollViewExample> createState() =>
+      NestedScrollViewExampleState();
```

Comment:
In the past I've asked for the entire arrow function to be on 1 line, but given that we're about to adopt a [new formatter](https://github.com/dart-lang/dart_style/blob/main/CHANGELOG.md#300-wip), it's not going to matter soon 🙂

Reply:
I fixed it just in case!